### PR TITLE
Update Terraform providers

### PR DESCRIPTION
**Pull request by smneal**
Update the Terraform providers for cloudfoundry and statuscake so they work on darwin_arm64 processors. These are the latest macs used by several devs across the service.

### Checklist

- [ ] Make sure all information from the Trello card is in here
- [x] Attach to Trello card
- [x] Rebased main
- [x] Cleaned commit history
- [ ] Tested by running locally

### CODE REVIEW

**Comment by smneal** [comment]
[Update terraform providers](https://trello.com/c/7OBfhG1X/634-update-terraform-providers)

File: terraform/modules/statuscake/resources.tf [hunk 1]
```diff
@@ -1,12 +1,73 @@
...
+
+#   website_name   = each.value.website_name
+#   website_url    = each.value.website_url
+#   test_type      = each.value.test_type
+#   check_rate     = each.value.check_rate
+#   contact_group  = each.value.contact_group
+#   trigger_rate   = each.value.trigger_rate
+#   node_locations = each.value.node_locations
+#   confirmations  = 2
+# }
```
Comment (RMcVelia) [inline terraform/modules/statuscake/resources.tf:12]: comments can be removed

File: terraform/workspace_variables/production.tfvars.json [hunk 2]
```diff
@@ -19,27 +19,21 @@
           "website_url": "https://api.publish-teacher-training-courses.service.gov.uk/ping",
           "test_type": "HTTP",
```
Comment (RMcVelia) [inline terraform/workspace_variables/production.tfvars.json:20]: test_type can be removed in each tfvars

**Review by RMcVelia — APPROVED** [review]

File: terraform/modules/statuscake/resources.tf (hunk 1 above)
Comment (smneal) [reply terraform/modules/statuscake/resources.tf]: Removed

File: terraform/workspace_variables/production.tfvars.json (hunk 2 above)
Comment (smneal) [reply terraform/workspace_variables/production.tfvars.json]: Removed

**Review by defong — APPROVED** [review]

**Review by RMcVelia — APPROVED** [review]

**Pull request merged**